src/parseexcel.py

```diff
@@ -0,0 +1,54 @@
+
+pptmimeList = ["application/vnd.ms-powerpoint","application/vnd.openxmlformats-officedocument.presentationml.presentation"]
+
+outfile = open("links.json","w")
+
+for i in range(len(x.index)):
+    
+    ID = x.loc[i]["id"]
```

Comment:
nit: use snake case for variables
```suggestion
    id = x.loc[i]["id"]
```